Components/Form.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useEffect } from "react";
+
+function Form() {
+  const [todo, setTodo] = useState("");
+  const [todoList, setTodoList] = useState([]);
+  let counter = 0;
+  
+  const handleChange = (e) => {
+    setTodo(e.target.value);
+  };
+
+  const addTodo = () => {
+    if (todo != " ") {
```

Comment:
What's the reasoning behind this? It seems like that this allows empty strings `""` to be added as todos. When you reset the input, you can reset it to an empty string `""`